usage/launcher/src/test/java/org/apache/brooklyn/launcher/BrooklynWebServerTest.java

```diff
@@ -146,11 +146,13 @@ public void verifyHttpsFromConfig() throws Exception {
     @Test
     public void verifyHttpsCiphers() throws Exception {
         brooklynProperties.put(BrooklynWebConfig.HTTPS_REQUIRED, true);
-        brooklynProperties.put(BrooklynWebConfig.TRANSPORT_PROTOCOLS, "XXX");
-        brooklynProperties.put(BrooklynWebConfig.TRANSPORT_CIPHERS, "XXX");
```

Comment:
I added the expected exception in the comments within the code: `javax.net.ssl.SSLHandshakeException` due to failure to verify the server certificate.